tests/test_cold_demand.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+from unittest import TestCase
+
+from mesido.esdl.esdl_parser import ESDLFileParser
+from mesido.esdl.profile_parser import ProfileReaderFromFile
+
+from rtctools.util import run_optimization_problem
+
+from utils_tests import demand_matching_test, energy_conservation_test, heat_to_discharge_test
+
+
+class TestColdDemand(TestCase):
+    def test_cold_demand(self):
+        """
+        This is a single line ring model, meaning that there are no dedicated supply or return
+        lines. This means that this model pipes are not related (no relation between hot and cold
+        pipes exists).
```

Comment:
I do not fully understand this, this description seems to be about a different case

Reply:
done